docs/md/checkpoint_learn_build.md

```diff
@@ -0,0 +1,50 @@
+\page ckpt_learn_build How to Build
+
+checkpoint can be built with `cmake`.
+
+\section ckpt_how-to-build Building
+
+To build *checkpoint*, one must obtain the following dependencies:
+
+\subsection ckpt_required-deps Required
+  - [detector](https://github.com/DARMA-tasking/detector)
+
+\subsection ckpt_optional-deps Optional
+
+  - gtest,   (if testing *checkpoint* is enabled)
+         - MPI,   (if MPI tests are enabled)
```

Comment:
random indent

Reply:
Fixed indent (but still keeping the sublist).